tests/pass-dep/tokio/mpsc.stdout

```diff
@@ -0,0 +1,2 @@
+GOT = sending from first handle
+GOT = sending from second handle
```

Comment:
This is not deterministic, is it? A different interleaving of threads can result in a different output and thus make this test fail.

Reply:
Right, I just change the output instead, so even if it has a different interleaving, the test still won't fail.